reporter/html.js

```diff
@@ -723,6 +723,20 @@ QUnit.testDone(function( details ) {
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
+
+	// Only add this information if stackTrace is supported
+	if ( Error().stack ) {
```

Comment:
details.source can be made by more kinds of implementation, I didn't want to bring the complexity from core's [extractStacktrace](https://github.com/jquery/qunit/blob/master/src/core.js#L474-L516) to test it, neither change that method causing a backwards incompatibility.